cartography/intel/github/repos.py

```diff
@@ -50,6 +50,16 @@
                         login
                         __typename
                     }
+                    collaborators(affiliation: OUTSIDE, first: 100) {
```

Comment:
I am making an assumption that a single repo will have 100 or fewer outside contributors. I also don't want to re-write this query to do dual pagination :(